services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/oauth/OAuthManagerImplUnitTest.java

```diff
@@ -18,14 +18,16 @@
 	
 	private OAuthManagerImpl oauthManager;
 	private OAuthProviderBinding providerBinding;
-	private OAuthProvider PROVIDER_ENUM = OAuthProvider.ORCID;
+	private OAuthProvider ORCID_PROVIDER_ENUM = OAuthProvider.ORCID;
+	private OAuthProvider NIH_PROVIDER_ENUM = OAuthProvider.NIH_OAUTH_2_0;
 
 	@Before
 	public void before() throws Exception {
 		oauthManager = new OAuthManagerImpl();
 		Map<OAuthProvider, OAuthProviderBinding> providerMap = new HashMap<OAuthProvider, OAuthProviderBinding>();
 		providerBinding = Mockito.mock(OAuthProviderBinding.class);
-		providerMap.put(PROVIDER_ENUM, providerBinding);
+		providerMap.put(ORCID_PROVIDER_ENUM, providerBinding);
+		providerMap.put(NIH_PROVIDER_ENUM, providerBinding); // TODO: does this need to be a different provider binding?
```

Comment:
Yes, it does.  Because then you can test that the expected provider binding (e.g., NIHOAuth2Provider) is the one retrieved.